net-analyzer/sn0int/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>git@rxv.cc</email>
+        <name>kpcyrd</name>
+    </maintainer>
+    <upstream>
+        <remote-id type="github">kpcyrd/sn0int</remote-id>
+    </upstream>
+    <use>
+        <flag name="man">Build and install man pages.</flag>
```

Comment:
This should be globally defined already.

Reply:
This was failing my build at first, I had to copy this line from https://github.com/gentoo/gentoo/blob/ac10ebbe3ecde46f95676d935147bc4b6271222b/sys-apps/ripgrep/metadata.xml

Reply:
it's ok as is, it's still a local flag